core/src/main/scala/com/whisk/docker/DockerContainer.scala

```diff
@@ -19,4 +20,6 @@ case class DockerContainer(
   def withReadyChecker(checker: DockerReadyChecker) = copy(readyChecker = checker)
 
   def withEnv(env: String*) = copy(env = env)
+
+  def withVolumes(volumeMappings: (String, (String, Boolean))*) = copy(volumeMaps = volumeMappings.toMap)
```

Comment:
can we define some type for volume mapping. I barely can infer semantic of this Tuple without reading rest of the code